src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_doc_library.baseline

```diff
@@ -137,7 +137,7 @@ describe('LibraryServiceSmokeTest', () => {
     });
 
     var formattedName = client.bookPath("testShelf-" + Date.now().toString(), projectId);
-    var rating = library.v1.types.Book.Rating.GOOD;
+    var rating = 'GOOD';
```

Comment:
Why are we nixing the types?

Reply:
@landtiro pointed out that there is no way to properly merge types from multiple sources together and avoid collisions.

We are importing types from:

  * the API
  * longrunning
  * IAM
  * common protos

And there could potentially be conflicts. The `types` module essentially compresses the various namespaces onto one spot. I _really_ like that (because the excessive namespacing is a confusing point for users) but our documentation is sourced from the `doc_` files, so the types module seems superfluous.

I am okay with having it, though, if it is valuable to users.

Reply:
The `types` wasn't being used for docs, but rather was user-accessible. It's useful because they don't need to know the magic strings/numbers, they can just access the generated type, e.g. `library.v1.types.Book.Rating.GOOD` instead of `'GOOD'`.

I vote that we keep the types, but un-squash them. When the user accesses them they should follow the path as it would be in the protos. We really should give users access to the stuff loaded by Protobuf.js

Reply:
Merging this as-is for now (this is an additive change so we do not need to block on it, and we have other repos waiting on this).

In principle, I am willing to do this, but I want to make sure we think through the strategy. I do not love `types.google.cloud.language.v1.Document` as something we ask a user to put in their code, for example, and it also introduces inconsistency in nomenclature (e.g. `video-intelligence` vs. `videointelligence`).

We can circle back and find the right thing,